docs/upgrading_to_v3.0.md

```diff
@@ -0,0 +1,134 @@
+# Upgrade Guidance
+Before moving forward with adopting components of v3 review the list of breaking changes below. You can find a complete list of features, bug fixes and other updates in the [Changelog](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/CHANGELOG.md).
+
+**Important:** There is no in-place upgrade path from v2 to v3.
+
+## Breaking Changes
+
+- Upgrade minimum required Terraform version to 1.3.0, previously was 0.13.7.
+- Added usage of Remote State data. This feature centralizes terraform variables that are input through all steps in a remote state in 0-bootstrap step. This uses a Google Storage bucket so that you will not have the need to re-declare them while deploying the foundation example. Note that each step requires `read/write` access to this bucket in order to be properly deployed.
```

Comment:
nit: this is an implementation detail which I don't think we need to call out
```suggestion
```

Reply:
ok